docs/source/markdown/podman-run.1.md.in

```diff
@@ -577,6 +577,8 @@ $ podman run -v data:/data2 -i -t fedora bash
 $ podman run -v /var/cache/dnf:/var/cache/dnf:O -ti fedora dnf -y update
 
 $ podman run -d -e MYSQL_ROOT_PASSWORD=root --user mysql --userns=keep-id -v ~/data:/var/lib/mysql:z,U mariadb
+
+$ podman run -d -e MYSQL_ROOT_PASSWORD=root --user mysql --userns=keep-id:uid=999,gid=999 -v ~/data:/var/lib/mysql:z mariadb
```

Comment:
A comment above elaborating on the user and group would be helpful.

Reply:
Done